### DISCUSSION TITLE
how to make newline in MathJax

### DISCUSSION THREAD

**Discussion by EthanMaDev** [mkdocs/mkdocs#3420] [Q&A] [answered]
I just start to use MathJax, put my markdown to mkdocs. but the newline in markdown is not ok

for example:
$$
(Y_i(0),Y_i(1),X_i,W_i) \sim P \\\\
ATE := E[Y(1) - Y(0)] 
$$
 
and the result like
<img width="423" alt="image" src="https://github.com/mkdocs/mkdocs/assets/12853615/b1ee1746-870f-47ef-bf9f-3c4cb655d440">

**Comment by facelessuser** [comment]
You can do the following:

```
$$
\displaylines{
(Y_i(0),Y_i(1),X_i,W_i) \sim P \\
ATE := E[Y(1) - Y(0)]
}
$$
```

For some reason, `\\` isn't implemented in MathJax 3 to work as you expect (yet?). But using `\displaylines{}` works.

**Comment by facelessuser** [comment, marked as answer]
This also works

```
$$
\begin{align}
(Y_i(0),Y_i(1),X_i,W_i) \sim P \\
ATE := E[Y(1) - Y(0)]
\end{align}
$$
```

**Comment by redtide** [comment]
~~Found this discussion and I'm not sure if to open a new one, which is exactly about the opposite situation, which wasn't happening when using Jekyll Any ideas?~~

EDIT: never mind, solved in #3439
